src/routes/github/github-router.ts

```diff
@@ -53,11 +53,11 @@ subRouter.post("/webhooks",
 
 // Create-branch is seperated above since it currently relies on query param to extract the jirahost
 // Todo able to move under the jirasymmetric middleware once flag completed
-subRouter.use("/create-branch", JiraHostFromQueryParamMiddleware, maybeJiraSymmetricJwtMiddleware, GithubServerAppMiddleware, GithubAuthMiddleware, csrfMiddleware, GithubCreateBranchRouter);
+subRouter.use("/create-branch", JiraHostFromQueryParamMiddleware, maybeJiraSymmetricJwtMiddleware, GithubServerAppMiddleware, csrfMiddleware, GithubCreateBranchRouter);
 
-subRouter.use("/repository", JiraHostFromQueryParamMiddleware, GithubServerAppMiddleware, GithubAuthMiddleware, csrfMiddleware, GithubRepositoryRouter);
+subRouter.use("/repository", JiraHostFromQueryParamMiddleware, GithubServerAppMiddleware, csrfMiddleware, GithubRepositoryRouter);
 
-subRouter.use("/branch", JiraHostFromQueryParamMiddleware, GithubServerAppMiddleware, GithubAuthMiddleware, csrfMiddleware, GithubBranchRouter);
+subRouter.use("/branch", JiraHostFromQueryParamMiddleware, GithubServerAppMiddleware, csrfMiddleware, GithubBranchRouter);
 
```

Comment:
Now these routers don't require OAuth token, I think, we should move create-branch routes to a different router; seperate it from github router.

Reply:
Moreover, we should also need to revisit the route pattern for` /repository` and ` /branch`. For instance; `getBranches` route inside `/create-branch` wheres as `getBranch` route inside `/github` route

Reply:
Agree on the route move, plus yeah, probably worth moving to create-branch specific routes.

I will do this as part of the "clean up" PR after this one. Need to remove all the refs to tenantHost and jiraHost plus all the conditional middleware 😢